spec/mongoid/query_cache_spec.rb

```diff
@@ -458,6 +458,19 @@
       end
     end
 
+   context 'when querying colleciton larger than the batch size' do
+     before do
+       101.times { Band.create! }
+     end
+
+     it 'does not raise an exception when querying multiple times' do
+       expect do
+         Band.all.to_a
```

Comment:
Can these also assert at least the length of arrays returned and ideally contents?

Reply:
Good call! Done.